prime-dotnet-webapi/Controllers/EnrolleeAgreementsController.cs

```diff
@@ -153,5 +157,42 @@ public async Task<ActionResult<EnrolleeProfileVersion>> GetEnrolmentForAgreement
 
             return Ok(ApiResponse.Result(enrolleeProfileHistory));
         }
+
+        // GET: api/Enrollees/5/agreements/2/download-unsigned
+        /// <summary>
+        /// Downloads a specific unsigned access term for an enrollee.
+        /// </summary>
+        /// <param name="enrolleeId"></param>
+        /// <param name="agreementId"></param>
+        [HttpGet("{enrolleeId}/agreements/{agreementId}/download-unsigned", Name = nameof(GetAccessTermDownloadUnsigned))]
```

Comment:
On the Org agreement side, the URL we landed on for downloading the PDF was organizations/{id}/agreements/{id}/signable. Can we change "download-unsigned" => "signable"